.github/workflows/tests-integration.yml

```
@@ -14,6 +24,30 @@ on:
           - run-integration-tests-cf-env
           - run-integration-tests-cf-env-with-client-creds
           - run-cats-cf-env
+      nodes:
+        description: Number of test nodes
+        required: false
+        type: string
+        default: "8"
```

Same as in prior file
```suggestion
        default: "12"
```